### DISCUSSION TITLE
how to use Multi-Tab & fzf, zoxide

### DISCUSSION THREAD

**Discussion by fastfading** [ajeetdsouza/zoxide#774] [Q&A] [answered]
there are many feature such as Multi-Tab & fzf, zoxide
mentioned in https://yazi-rs.github.io/features
however there is no doccument for it . 

new users like me are very confused. 
could you tell me how to use Multi-Tab & fzf, zoxide ?
1. how to create / delete new tab
2. how to use   fzf (Z), zoxide(z)
I have install zoxide ,  but got error   "zoxide: no match found"  when I hit z button 

there are

**Comment by fastfading** [comment]
since copy cross multiple yazi is not supported,    this multi-tab is very useful.

**Comment by ajeetdsouza** [comment, marked as answer]
Multi-tab is a yazi feature. This is the zoxide project, you should be asking for support on yazi's issue tracker: https://github.com/sxyazi/yazi/issues

Regarding their zoxide integration, try going through the README to understand how zoxide works: https://github.com/ajeetdsouza/zoxide
